nuntium/templates/nuntium/profiles/contacts/contacts-per-writeitinstance.html

```diff
@@ -38,24 +38,24 @@
   <div class="page-header">
     <h2>{% trans "Recipients" %}</h2>
   </div>
-
+  <p>{% trans "There are" %} <strong>{{ writeitinstance.persons.count }}</strong> {% trans "recipients in your WriteIt" %}</p>
```

Comment:
Yes, you can use [`{% blocktrans %}`](https://docs.djangoproject.com/en/1.6/topics/i18n/translation/#std:templatetag-blocktrans) to handle variables in language sentences and pluralization:

```
{% blocktrans with count person_count=writeitinstance.persons.count %}
There is 1 recipient in your WriteIt
{% plural %}
There are {{ person_count }} recipients in your WriteIt
{% endblocktrans %}
```